workflows/snakemake/Snakefile

```diff
@@ -0,0 +1,130 @@
+configfile: "./config.yaml"
+workdir: "/home/pkalverla1/Urban-M4/experiments/snakemake"
```

Comment:
Could this be added to the config file?

Reply:
Good idea, done in [121074b](https://github.com/Urban-M4/misc/pull/6/commits/121074bc83d80ea9944292b2ee50b8d40ab4a0b3)